baystation12.dme

```diff
@@ -745,6 +745,7 @@
 #include "code\game\machinery\vending.dm"
 #include "code\game\machinery\vending_deconstruction.dm"
 #include "code\game\machinery\vitals_monitor.dm"
+#include "code\game\machinery\vox.dm"
```

Comment:
Please change the file name from `code\game\machinery\Vox.dm` to `code\game\machinery\vox.dm"` if you are gonna change the include like that, it will mess up.